src/lib/functions/document/vw.ts

```diff
@@ -1,3 +1,3 @@
 export function vw(vw: number) {
-    return (vw * globalThis.window.innerWidth) / 100;
+    return (vw * globalThis.window?.innerWidth) / 100 ?? null;
```

Comment:
Returning `null`? uhh did you check on use case? maybe there should be type error ig.

Reply:
> Returning null? uhh did you check on use case? maybe there should be type error ig.

It's fine :) 

I did check tippy one. Since we are calculating at instance start, this case is good enough